crfsuite/fix-autoconf-automake.patch

```diff
@@ -0,0 +1,92 @@
+# PR is merged and not merged in master branch.
```

Comment:
Can you link the PR here, so we can tell if the patch is still needed?

Reply:
Ah good catch! Fixed. CI is now 🟢.